restapi.py

```diff
@@ -208,4 +254,4 @@ def get(self):
 # --------------------------------------
 
 if __name__ == '__main__':
-    app.run(host='0.0.0.0')
+    app.run(host='0.0.0.0', debug=True)
```

Comment:
Is it strictly needed to run the app in debug mode?

Reply:
Fixed in https://github.com/ibpsa/project1-boptest/pull/441/commits/1e93eb94ff86c40971f0ac86a25e002dc500f419.